pandas/core/frame.py

```diff
@@ -1088,7 +1088,11 @@ def itertuples(self, index=True, name="Pandas"):
         arrays.extend(self.iloc[:, k] for k in range(len(self.columns)))
 
         if name is not None:
-            itertuple = collections.namedtuple(name, fields, rename=True)
+            # https://github.com/python/mypy/issues/848
+            # error: namedtuple() expects a string literal as the first argument  [misc]
```

Comment:
cant do a cast or assert?

Reply:
hmm, I've got the wrong mypy issue here.

> cant do a cast or assert?

no, it has to be a string literal and doesn't look like mypy will change.

see https://github.com/python/mypy/issues/9046#issuecomment-649736524

we can remove this ignore with Final in Python 3.8 (and a later version of mypy)